dotnet/src/SemanticKernel.Skills/Skills.Document/Skills.Document.csproj

```diff
@@ -14,7 +14,7 @@
   <PropertyGroup>
     <!-- NuGet Package Settings -->
     <PackageId>Microsoft.SemanticKernel.Skills.Document</PackageId>
-    <Title>Semantic Kernel - OpenXml Connector</Title>
+    <Title>Semantic Kernel - Document Skill</Title>
```

Comment:
Not related but needed to be corrected

Reply:
@gitri-ms FYI in case this was intentional

Reply:
That looks like a holdover from when the OpenXml connector was separate from the Document skill. Thanks @glahaye for correcting.